cmd/job_run.go

```diff
@@ -73,11 +79,17 @@ func (cmd *JobRun) Execute(args []string) (err error) {
 	vols := map[string]*svc.JobVolume{}
 	for _, input := range cmd.Inputs {
 		parts := strings.Split(input, ":")
-		if len(parts) != 2 {
+		if len(parts) < 1 {
```

Comment:
Maybe add a error case when there are more then 2 colons, this is probably always an error on the users side.